src/TestFramework/TestFramework/Resources/xlf/FrameworkMessages.cs.xlf

```diff
@@ -283,6 +283,11 @@ Skutečnost: {2}</target>
         <target state="translated">Rozdílný počet elementů.</target>
         <note></note>
       </trans-unit>
+      <trans-unit id="ReturnedSubsetValueMessage">
+        <source>Element(s) &lt;{0}&gt; is/are not a subset of the superset</source>
```

Comment:
```suggestion
        <source>Element(s) &lt;{0}&gt; is/are not present in the collection</source>
```
Alternative wording. I find it hard to process what happened when thinking about subset of superset. but very easy to understand that the values I was looking for were not in the collection.